git push 的时候出错branch is currently checked out
```shell
# 在仓库里执行
git config receive.denyCurrentBranch ignore
```